modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Detail.cshtml

```diff
@@ -230,6 +230,7 @@
                                                 <div>
                                                     <form class="editFormClass">
                                                         <input name="commentId" value="@commentWithRepliesDto.Comment.Id" hidden />
+                                                        <input name="concurrencyStamp" value="@commentWithRepliesDto.Comment.ConcurrencyStamp" hidden />
```

Comment:
I've never seed `<input hidden />` usage like that. Is it working? 
It should [normally](https://www.w3schools.com/tags/att_input_type_hidden.asp) be `<input type="hidden" />`

Reply:
I've tested, it works but it's better to use type="hidden" to prevent compatibility issues, I'll change it

Reply:
Resolved with commit c7050e9ff2641de4f55981c4ebad39cc61d63870